MapleServer2/Packets/FieldObjectPacket.cs

```diff
@@ -90,6 +90,17 @@ public static Packet LoadNpc(IFieldObject<Npc> npc)
                 .Write(npc.Coord);
         }
 
+        public static Packet LoadMob(IFieldObject<Mob> mob)
+        {
+            return PacketWriter.Of(SendOp.PROXY_GAME_OBJ)
+                .WriteByte(0x6)
+                .WriteInt(mob.ObjectId)
+                .WriteInt(mob.Value.Id)
+                .WriteByte()
+                .WriteInt(200)
+                .Write(mob.Coord);
```

Comment:
Prefer non-chaining method calls for writing packets, makes it easier to organize and make changes.